routes/favicon.coffee

```diff
@@ -0,0 +1,28 @@
+request = require 'request'
+fs = require 'fs'
+cachefile = "favicon.ico"
```

Comment:
I have few experience in coffee script, but this seems natural, coming from Python.